modules/images/image-loading-optimization/detection.php

```diff
@@ -35,6 +35,9 @@ function ilo_get_detection_script( string $slug, array $needed_minimum_viewport_
 	 */
 	$detection_time_window = apply_filters( 'ilo_detection_time_window', 5000 );
 
+	$web_vitals_lib_data = require_once __DIR__ . '/detection/web-vitals.asset.php';
```

Comment:
This shouldn't be `require_once` because then if this function gets called again, it won't return anything.

See https://github.com/WordPress/gutenberg/blob/970488826c004014862b3f92458e102db2201d88/lib/client-assets.php#L198

```suggestion
	$web_vitals_lib_data = require __DIR__ . '/detection/web-vitals.asset.php';
```